source/tests/system/nirfmxbt_driver_api_tests.cpp

```diff
@@ -18,13 +18,20 @@ namespace system {
 namespace {
 
 const auto PXI_5663E = "5663E";
+const int kPreambleSyncPacketStartDetectionFailedWarning = 686280;
```

Comment:
nit: in files that use UPPER_SNAKE for constants, let's stick with that.

Reply:
Retracting, you can use whichever you want. As Ryan pointed out to me, this is what the driver APIs tend to use so it's reasonable to mimic them.

Reply:
Either way, the two constants in my file weren't consistent, so I went back and changed `PXI_5663E` to `kPxi5663e`.